src/coreclr/vm/stubgen.cpp

```diff
@@ -109,7 +109,7 @@ void ILStubLinker::DumpIL_FormatToken(mdToken token, SString &strTokenFormatting
             SString typeNameUtf8;
             typeName.ConvertToUTF8(typeNameUtf8);
             SString strFieldName(SString::Utf8, pFD->GetName());
-            strTokenFormatting.Printf("%s::%s", typeNameUtf8.GetUTF8NoConvert(), strFieldName.GetUTF8NoConvert());
+            strTokenFormatting.Printf("%s::%s", typeNameUtf8.GetUTF8(), strFieldName.GetUTF8());
```

Comment:
```suggestion
            strTokenFormatting.Printf("%s::%s", typeName.GetUTF8(), pFD->GetName());
```
And the 3 lines above can be deleted